src/App.js

```diff
@@ -9,32 +10,39 @@ import { BrowserRouter as Router } from "react-router-dom";
  * Footer: Global Footer
  * Navbar: Global navigation bar
  */
-import {
-  Footer,
-  Navbar,
-} from "./components/molecules";
-// Routes
+import { Footer, Navbar } from "./components/molecules";
+// Starts the page on top when reloaded or redirected
+import { ScrollToTop } from "./components/atoms";
+
+//> Routes
 import Routes from "./Routes";
+//#endregion
 
+//#region > Components
 class App extends React.Component {
   render() {
     return (
       <Router>
-        <div className="flyout">
-          <Navbar />
-          <main>
-            <Routes />
-          </main>
-          <Footer />
-        </div>
+        <ScrollToTop>
+          <div className="flyout">
+            <Navbar />
+            <main>
+              <Routes />
+            </main>
+            <Footer />
+          </div>
+        </ScrollToTop>
       </Router>
     );
   }
 }
+//#endregion
 
+//#region > Exports
 export default App;
```

Comment:
In SNEK we added comments, to define the export. If this is to much overhead ignore it.
```suggestion
//#region > Exports
//> Default Class
export default App;
```

Reply:
Too much overhead, yes.